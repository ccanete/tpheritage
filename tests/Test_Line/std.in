L test 2 4 4 8
EXIT