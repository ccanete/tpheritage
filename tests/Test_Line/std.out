OK
# Line test passing by (2, 4) and (4, 8) has been created.

# The program is being closed.